common-moonmage.lic

```diff
@@ -54,7 +54,7 @@ module DRCMM
   end
 
   def center_telescope(target)
-    DRC.bput("center telescope on #{target}", 'You put your eye', 'open it to make any use of it', 'The pain is too much', "That's a bit tough to do when you can't see the sky", 'Your search for')
+    DRC.bput("center telescope on #{target}", 'Center what?', 'You put your eye', 'open it to make any use of it', 'The pain is too much', "That's a bit tough to do when you can't see the sky", 'Your search for')
```

Comment:
Center what?

Reply:
@rpherbig

Reply:
Did you test that this works?

For some reason I have it in my head that `bput` interprets the match message as a regex, so the `?` in the message gives me pause. Will it be interpreted as the literal `?` or as "Match between 0 and 1 of the preceding token."?

Reply:
I just removed it, no reason to include it anyway.

Reply:
Good call.